src/client/hooks/useNotifications.ts

```diff
@@ -15,6 +16,7 @@ const appStateNotificationsSelector = selectorFamily({
     (profileType: ProfileType) =>
     ({ get }) => {
       const appState = get(appStateAtom);
+      const isLoggedInAsCompany = get(sessionAtom).userType === 'BEDRIJF';
```

Comment:
Hier kun je `profileType` voor gebruiken. In dit geval: `profileType === 'commercial'`